internal/config/specs.go

```diff
@@ -57,4 +57,10 @@ type EnvSpec struct {
 	PayloadValidationEnabled bool `envconfig:"payload_validation_enabled" default:"true"`
 
 	OpenFGAWorkersTotal int `envconfig:"openfga_workers_total" default:"150"`
+
+	MailHost        string `envconfig:"MAIL_HOST" required:"true"`
```

Comment:
Just have a question about the new env vars:

When it comes to the admin UI charm, how do we like to pass the values to these env vars? From the integration data (e.g. integration with a certain SMTP charm) or from juju configs?

Reply:
I was thinking we would do it trhough integration data with an smtp charm
What do you think @shipperizer ?

Reply:
yes i think integration is the right way to go